project.yml

```diff
@@ -5,7 +5,7 @@ packages:
   SpruceIDMobileSdk:
     url: https://github.com/spruceid/mobile-sdk-swift
     # from: 0.0.10
-    revision: 8b80370
+    revision: 379b4a3d4550df6ab9da0e5be3406d8021dcd253
```

Comment:
Pointing to the last commit on https://github.com/spruceid/mobile-sdk-swift/pull/34